From 5047570120a3b56354485a51ccf7e578fad3a230 Mon Sep 17 00:00:00 2001
From: Anurag Kumar Vulisha <anurag.kumar.vulisha@xilinx.com>
Date: Wed, 21 Sep 2016 19:07:24 +0530
Subject: [PATCH 205/223] devicetree: dwc3: Add LPM transfers support on zcu102
 board

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

This patch adds USB 3.0 LPM transfers suuport to zcu102
board by adding snps,usb3_lpm_capable property.

Signed-off-by: Anurag Kumar Vulisha <anuragku@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 4e9400a7beb0297d57f69fb9fddaadee22c502f8)
---
 arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts
index 401a453..8644d9a 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102.dts
@@ -629,6 +629,7 @@ drivers/hwmon/pmbus/Makefile:11:obj-$(CONFIG_SENSORS_MAX20751)  += max20751.o
 &dwc3_0 {
 	status = "okay";
 	dr_mode = "host";
+	snps,usb3_lpm_capable;
 	phy-names = "usb3-phy";
 	phys = <&lane2 PHY_TYPE_USB3 0 2 26000000>;
 };
-- 
2.7.4

